src/host/tracing.hpp

```diff
@@ -17,28 +17,29 @@ Author(s):
 
 #pragma once
 
-#include <functional>
-
 #include "../types/inc/Viewport.hpp"
 
-#if DBG
-#define DBGCHARS(_params_)              \
-    {                                   \
-        Tracing::s_TraceChars _params_; \
-    }
-#define DBGOUTPUT(_params_)              \
-    {                                    \
-        Tracing::s_TraceOutput _params_; \
-    }
-#else
-#define DBGCHARS(_params_)
-#define DBGOUTPUT(_params_)
-#endif
-
-#define TraceLoggingConsoleCoord(value, name) \
-    TraceLoggingStruct(2, name),              \
-        TraceLoggingInt32(value.X, "X"),      \
-        TraceLoggingInt32(value.Y, "Y")
+#define TraceLoggingConsoleCoord(value, name)        \
+    TraceLoggingPackedData(&value, sizeof(COORD)),   \
+        TraceLoggingPackedStruct(2, name),           \

```

Comment:
what's the difference - why packed?

Reply:
The TraceLogging documentation (in the header file) says to use this, if one has packed struct fields, because it saves space and improves performance.